### PR TITLE
VP-1452 - New signups are now assigned the Voluntarily organisation

### PR DESCRIPTION
**Pull request by RoughChopTom**
## I do solemnly swear that I have:
- [ ] Run `npm test` and all the tests pass.
- [ ] Run `npm run lint` and there are no warnings.
- [ ] Not decreased the overall test coverage?
- [ ] Included the Jira ID and description in the PR title

## Proposed Changes? 🤔
1. New signups are assigned the Voluntarily organisation
2. Do not send the member update email if the organisation is assigned via the system.
3. 

## Additional Info.🧐

Any additional info goes here

### CODE REVIEW

File: server/api/person/person.subscribe.js [hunk 1]
```diff
@@ -28,10 +33,28 @@ module.exports = (server) => {
...
 
+  PubSub.subscribe(TOPIC_PERSON__CREATE, async (msg, person) => {
+    const coreOrganisation = await Organisation.findOne({ name: CORE_ORGANISATION_NAME }).lean().exec()
+
+    if (!coreOrganisation) {
+      return
+    }
+
+    await addMember({
+      person: person.id.toString(),
```
Comment (avowkind) [inline server/api/person/person.subscribe.js]: I wonder whether this should be person._id?

person.id would likely return null and fail to save the member record.

File: server/api/person/person.subscribe.js [hunk 2]
```diff
@@ -115,3 +138,7 @@ PubSub.subscribe(TOPIC_INTEREST__MESSAGE, async (msg, interest) => {
   await info
   PubSub.publish(TOPIC_PERSON__EMAIL_SENT, info)
 })
+
+function isSystemOperation (validationMessage) {
```
Comment (avowkind) [inline server/api/person/person.subscribe.js]: suggest replacing with.
const isSystemOperation = validationMessage => validationMessage === SYSTEM_MEMBER_VALIDATION_MESSAGE

File: server/api/person/person.subscribe.js [hunk 3]
```diff
@@ -28,10 +33,28 @@ module.exports = (server) => {
...
+      organisation: coreOrganisation._id.toString(),
+      validation: SYSTEM_MEMBER_VALIDATION_MESSAGE
+    })
+  })
+
   PubSub.subscribe(TOPIC_MEMBER__UPDATE, async (msg, member) => {
     // a new member has been created or a member status has changed
     // send email to let people know
 
+    if (isSystemOperation(member.validation)) {
```
Comment (avowkind) [inline server/api/person/person.subscribe.js]: function not really needed here
if (member.validation === SYSTEM_MEMBER_VALIDATION_MESSAGE)) return 

would suffice

File: server/api/person/person.subscribe.js (hunk 1 above)
Comment (RoughChopTom) [reply server/api/person/person.subscribe.js]: Have updated to `person._id.toSTring()`

File: server/api/person/person.subscribe.js (hunk 3 above)
Comment (RoughChopTom) [reply server/api/person/person.subscribe.js]: Done

File: server/api/person/person.subscribe.js (hunk 2 above)
Comment (RoughChopTom) [reply server/api/person/person.subscribe.js]: I have removed this function.

**Pull request merged**